File: poo/java/Entrada/Entrada.java
/*
 *  Programa escrito com propósito didático apenas.
 *  @author Nathan Cruz <nathan.cruz@unifesp.br>
 *  @created 28/03/14
 *  R.A.: 86492
 */

import java.util.*;

public class Entrada{
    public static void main(String args[]){
        Scanner entrada = new Scanner(System.in);
        String s;
        Triangulo tri;
        Retangulo ret;
        float n1, n2, n3;

        System.out.println("O que deseja criar?[R/T]");
        s = entrada.next();
        if(s.toUpperCase() == "R"){
            System.out.println("Entre com a base: ");
            n1 = (float) entrada.nextDouble();
            System.out.println("Entre com a altura: ");
            n2 = (float) entrada.nextDouble();

            ret = new Retangulo(n1, n2);
            System.out.println("Area: " + ret.Area());
            System.out.println("Perimetro: " + ret.Perimetro());
        }
        else if(s.toUpperCase() == "T"){
            System.out.println("Entre com o primeiro lado: ");
            n1 = (float) entrada.nextDouble();
            System.out.println("Entre com o segundo lado: ");
            n2 = (float) entrada.nextDouble();
            System.out.println("Entre com o terceciro lado: ");
            n3 = (float) entrada.nextDouble();

            tri = new Triangulo(n1, n2, n3);
            System.out.println("Area: " + tri.Area());
            System.out.println("Perimetro: " + tri.Perimetro());
            tri.tipo();
        }
        else{
        }
    }
}

File: poo/python/Entrada/Console/Entrada.py
#!/usr/bin/env python
#-*- coding: utf-8 -*-

"""
@author Nathan Cruz <nathan.cruz@unifesp.br>
@created 29/03/14
"""
from __future__ import unicode_literals
from __future__ import print_function

from Retangulo import Retangulo
from Triangulo import Triangulo

opcao = raw_input('O que deseja criar? [R/T] ')
if opcao.upper() == 'R':
    base = float(raw_input('Entre com a base: '))
    altura = float(raw_input('Entre com a altura: '))
    
    try:
        ret = Retangulo(base, altura)
        print('Area: %.2f' % (ret.Area()))
        print('Perimetro: %.2f' % (ret.Perimetro()))
    except ValueError:
        print('Os valores inseridos não compõe um retângulo')

elif opcao.upper() == 'T':
    lado1 = float(raw_input('Entre com o primeiro lado: '))
    lado2 = float(raw_input('Entre com o segundo lado: '))
    lado3 = float(raw_input('Entre com o terceiro lado: '))

    try:
        tri = Triangulo(lado1, lado2, lado3)
        print('Area: %.2f' % (tri.Area()))
        print('Perimetro: %.2f' % (tri.Perimetro()))
        tri.Tipo()
    except ValueError:
        print('Os valores inseridos não compõe um triângulo')
else:
    print('Valor invalido! Favor entrar com R ou Q')
